examples/bq-email-exports/export_query_results_function/main.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Cloud Function for exporting BigQuery results from an anonymous table to GCS.
+Triggered after BigQuery query is complete.
+"""
+
+import base64
+import json
+import logging
+import os
+
+from google.cloud import bigquery
+
+
+def main(event, context):
+    """Entrypoint for Cloud Function"""
+
+    data = base64.b64decode(event['data'])
+    pubsub_message = json.loads(data)
+    error = pubsub_message.get('errorStatus')
+    if error:
+        logging.error(RuntimeError(f"Error in upstream query job:{error}"))
```

Comment:
The in-built error message from the object returns the job id. The error block looks like this:
`{'code': 5, 'message': 'Not found: Dataset nehanene-dev:tf_test; JobID: nehanene-dev:5fb3f4cc-0000-2d86-97ed-94eb2c042718'}`

I'll print out the error message instead of the whole error block for prettiness though.